EventsExpress/ClientApp/src/components/profile/editProfile/ChangePassword.jsx

```diff
@@ -18,14 +18,14 @@ const renderTextField = ({
     meta: { touched, error },
     ...custom
 }) => (
-        <TextField
-            hintText={label}
-            floatingLabelText={label}
-            errorText={touched && error}
-            {...input}
-            {...custom}
-        />
-    );
+    <TextField
```

Comment:
I believe we have the same field-renderer in our helpers. Let's use that one. Or just move this one into a separate component inside the form-helpers module, so we will start using it instead of the terrible helpers' variant